server/src/models/Match.ts

```diff
@@ -0,0 +1,76 @@
+import { getSettings } from '@src/services/SettingsService'
+
+export class Match {
+  constructor (
+    public matchKey: string,
+    public eventKey: string,
+    public compLevel: 'qm' | 'sf' | 'f',
+    public setNumber: number,
+    public matchNumber: number,
+
+    public startTime: EpochTimeStamp,
+    public endTime: EpochTimeStamp,
```

Comment:
I would prefer if we use [Luxon](https://moment.github.io/luxon/#/) for datetime handling. It's considerably easier to parse and work with dates. Though in the model itself, we may or may not desire to store dates in either an ISO8601 string or epoch time.